ocaml/xapi/create_misc.mli

```diff
@@ -44,3 +44,4 @@ val create_host_cpu : __context:Context.t -> host_info -> unit
 val create_pool_cpuinfo : __context:Context.t -> unit
 val create_chipset_info : __context:Context.t -> host_info -> unit
 val create_updates_requiring_reboot_info : __context:Context.t -> host:[`host] Ref.t -> unit
+val get_xapi_verstring : unit -> string
```

Comment:
`get_xapi_verstring` - the fact that this is a string is covered by the type, so I would prefer `get_xapi_version`. This will lead to clearer code below because "version" will be used more consistently.